vllm_ascend/ops/rotary_embedding.py

```diff
@@ -18,7 +18,7 @@
 from typing import Optional, Tuple
 
 import torch
-from vllm.model_executor.layers.rotary_embedding import RotaryEmbedding
+from vllm.model_executor.layers.rotary_embedding import RotaryEmbedding ,DeepseekScalingRotaryEmbedding
```

Comment:
please aware of the coding style.
import RotaryEmbedding, DeepseekScalingRotaryEmbedding